apiserver/facades/controller/caasoperatorprovisioner/state.go

```diff
@@ -15,13 +15,15 @@ import (
 // CAASOperatorProvisionerState provides the subset of global state
 // required by the CAAS operator provisioner facade.
 type CAASOperatorProvisionerState interface {
+	network.SpaceLookup
```

Comment:
State indirections in the API server will include `SpaceLookup` where address conversion is required.